.github/workflows/wpt.yml

```
@@ -114,7 +114,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
         with:
-          node-version: 16
+          node-version: 18
```

Make it use latest LTS with this
```suggestion
          node-version: lts/*
```

---

Obsolete.